core/src/main/java/com/opensymphony/xwork2/interceptor/ChainingInterceptor.java

```diff
@@ -160,7 +160,20 @@ private void copyStack(ActionInvocation invocation, CompoundRoot root) {
         Map<String, Object> ctxMap = invocation.getInvocationContext().getContextMap();
         for (Object object : list) {
             if (shouldCopy(object)) {
-                reflectionProvider.copy(object, invocation.getAction(), ctxMap, prepareExcludes(), includes);
+                String actionConfigClassName = invocation.getProxy().getConfig().getClassName();
```

Comment:
Please wait and do not merge! until I check if `getConfig` and `getClassName` are always ready and are as expected e.g. when using convention or spring plugin!